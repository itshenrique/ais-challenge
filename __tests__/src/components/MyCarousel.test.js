/**
 * @format
 */
import 'react-native';
import React from 'react';
import Carousel from '../../../src/components/MyCarousel';
import {shallow} from 'enzyme';

describe('Carousel Tests', () => {
  it('render carousel', () => {
    const component = shallow(<Carousel />);
    expect(component).toMatchSnapshot();
  });
});
