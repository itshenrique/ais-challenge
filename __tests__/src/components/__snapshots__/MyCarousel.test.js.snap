// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Tests render carousel 1`] = `ShallowWrapper {}`;
